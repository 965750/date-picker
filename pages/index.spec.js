import {
  shallowMount,
  createLocalVue
} from '@vue/test-utils'
import index from './index'

const localVue = createLocalVue()

describe('index.vue', () => {
  const createWrapper = () => shallowMount(index, {
    localVue
  })

  it('should initial snapshot', () => {
    const wrapper = createWrapper()

    expect(wrapper.element).toMatchSnapshot()
  })
})
