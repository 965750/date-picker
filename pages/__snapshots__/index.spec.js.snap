// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.vue should initial snapshot 1`] = `
<div
  class="app"
>
  <dateform-stub />
</div>
`;
